REAMDE.md

```diff
@@ -0,0 +1,14 @@
+This document is built using Rust's
```

Comment:
The document is title REAMDE instead of README

Reply:
Ugh, sorry.